docs/source/pillars/data_management.md

```diff
@@ -149,6 +149,24 @@ The ability of the TRE operator to ensure outputs are safely published and share
     Results will be produced which address the project's purpose, some of which may be outputs that are removed from the TRE.
     Understanding what these outputs are likely to be and their sensitivity as early as possible will help prepare for their processing and publication.
   - Recommended
+* - 3.3.3.
+  - You must have a documented process for disclosure control of output files from the TRE.
+  - This process should define expected risks and how to mitigate them.
+    All TRE outputs must be subject to this process.
+    You might choose to follow existing guidelines, for example around statistical disclosure.
+  - Mandatory
+* - 3.3.4.
+  - You must have a process for assigning responsibility for output checking.
+  - A named person or team should be given responsibility for checking outputs.
+    This person or team must follow your disclosure control process and will be responsible for any automated parts of this process.
+    Output checking can help mitigate against unintentional data breaches during the egress procedure.
+  - Mandatory
+* - 3.3.5.
+  - Where the possibility exists that a project will require egress of outputs that cannot be manually checked, you must have a documented policy for handling disclosure risks associated with these outputs.
```

Comment:
```suggestion
  - You must have a documented policy for handling disclosure risks associated with outputs that cannot be manually checked.
```

Reply:
Is this too much for mandatory though? Could be wrong but I don't know if this would apply to all, or even most TREs - especially since it's arguably just bad practice

Reply:
But I think it should be mandatory conditional on the TRE operators/projects wanting to do it

Reply:
I think it can be mandatory- your policy can be that "outputs that can't be manually checked are always refused". If that's not clear then might be worgth adding?

Reply:
Added `any` to the suggested change.

Reply:
re-reading the guidance
> It is generally recommended that only files that can be checked are disclosed
I think that covers having a policy like "outputs that can't be manually checked are always refused"